.github/workflows/ci.yml

```diff
@@ -117,10 +118,9 @@ jobs:
       - name: Checkout Pex
         uses: actions/checkout@v3
       - name: Setup Python ${{ join(matrix.python-version, '.') }}
-        uses: gabrielfalcao/pyenv-action@v14
+        uses: jsirois/actions/pyenv@6b4b66ba805bdd553071b95d946467a64df55029
```

Comment:
I'll flip from jsirois to pantsbuild once https://github.com/pantsbuild/actions/pull/24 lands.

Reply:
Done.